[2024-11-07T07:18:04.194+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweatherdata_dag.task_2 manual__2024-11-07T07:17:58.544089+00:00 [queued]>
[2024-11-07T07:18:04.211+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweatherdata_dag.task_2 manual__2024-11-07T07:17:58.544089+00:00 [queued]>
[2024-11-07T07:18:04.212+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-11-07T07:18:04.242+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): task_2> on 2024-11-07 07:17:58.544089+00:00
[2024-11-07T07:18:04.261+0000] {standard_task_runner.py:60} INFO - Started process 5283 to run task
[2024-11-07T07:18:04.272+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'openweatherdata_dag', 'task_2', 'manual__2024-11-07T07:17:58.544089+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/model_meteo.py', '--cfg-path', '/tmp/tmpdyu4eer8']
[2024-11-07T07:18:04.274+0000] {standard_task_runner.py:88} INFO - Job 66: Subtask task_2
[2024-11-07T07:18:04.300+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-11-07T07:18:04.386+0000] {task_command.py:423} INFO - Running <TaskInstance: openweatherdata_dag.task_2 manual__2024-11-07T07:17:58.544089+00:00 [running]> on host 8a2ba9ce15fe
[2024-11-07T07:18:04.594+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweatherdata_dag' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2024-11-07T07:17:58.544089+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-07T07:17:58.544089+00:00'
[2024-11-07T07:18:04.606+0000] {logging_mixin.py:188} INFO -  Empty DataFrame
Columns: []
Index: []
[2024-11-07T07:18:04.617+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-11-07T07:18:04.638+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=openweatherdata_dag, task_id=task_2, execution_date=20241107T071758, start_date=20241107T071804, end_date=20241107T071804
[2024-11-07T07:18:04.733+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-07T07:18:04.790+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
